books/book.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="book.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Vollkorn:wght@500&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+    <script src="https://kit.fontawesome.com/a076d05399.js"></script>
+    <title>Book Details</title>
+  </head>
+  <body>
+    <header>
+      <div class="container">
+        <h1><a href="../index.html"> Books for all!</a></h1>
+      </div>
+    </header>
+    <main class="main-content"></main>
+
+    <footer>
+      <h5>&copy; 2020 Books For All, All Rights Reserved</h5>
```

Comment:
Try to fix this footer to the bottom of the page so it doesn't float in the middle of the page. This and other styling work can also be done in another PR if you have some more plans with this page

Reply:
I fixed the footer but can you tell me when does it float in middle of the page ? at which screen's size?

Reply:
![image](https://user-images.githubusercontent.com/33314617/102363801-0f67f100-3fb6-11eb-8a5c-03731fc7d20d.png)
 Here is a screenshot

Reply:
@josipgrgic  I think its fixed now

Reply:
![Capture](https://user-images.githubusercontent.com/47057524/102380560-59f26900-3fc8-11eb-80a6-66ffc1e6199f.PNG)